### PR TITLE
DAFA-AFC-2601-Review_order_interpreter

### PR DESCRIPTION
**Pull request by nils-vertel**
* Trivial cleanup fixes (spelling, line length etc)

### CODE REVIEW

**Comment by fredrikarvas-vertel** [comment]
@rupareliyahemangi145 could you please review this code and repond to @nils-vertel  when you are done.

**Comment by rupareliyahemangi145** [comment]
Yes Sure @FredrikArvas

File: outplacement_order_interpreter/__manifest__.py [hunk 1]
```diff
@@ -22,7 +22,8 @@
 # Version format OdooMajor.OdooMinor.Major.Minor.Patch
 {
     'name': 'Order Interpreter',
-    'version': '12.0.1.7.8',
+    'version': '12.0.1.7.3',
```
Comment (rupareliyahemangi145) [inline outplacement_order_interpreter/__manifest__.py]: @nils-vertel Here in this manifest file two lines for version added. There must be only one.

Comment (nils-vertel) [reply outplacement_order_interpreter/__manifest__.py]: Great catch, bad merge, fixed now.
Any other problems with the code?

Comment (rupareliyahemangi145) [reply outplacement_order_interpreter/__manifest__.py]: No @nils-vertel

**Pull request merged**